### DISCUSSION TITLE
xtracloud.net

### DISCUSSION THREAD

**Discussion by sami155** [symbuzzer/Turkish-Ad-Hosts#19] [Soru&Cevap / Q&A] [answered]
the following URLs are used as A-GPS from Qualcomm: 

path1.xtracloud.net
path2.xtracloud.net
path3.xtracloud.net 

More info:

The Qualcomm Location XTRA Service generates and provides accurate Satellite positions for extended periods of time to a mobile device. This assistance data is provided in compact form and allows the mobile device to perform positioning for an extended period of time (up to seven days). The assistance data files are made available to a Data Distribution Network for global distribution.

**Comment by symbuzzer** [comment, marked as answer]
![image](https://github.com/symbuzzer/Turkish-Ad-Hosts/assets/9841184/4d699e1f-998a-49dd-8aaa-6a76eb0227ab)

please look: https://restoreprivacy.com/qualcomm-denies-unlawful-user-location-data-collection-on-phones/
